python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -215,11 +240,11 @@ def determinePsf(
         stars = []
         for candidate in psfCandidateList:
             cmi = candidate.getMaskedImage()
-            weight = computeWeight(
-                cmi,
-                self.config.maxSNR,
-                self.config.zeroWeightMaskBits
-            )
+            good = getGoodPixels(cmi, self.config.zeroWeightMaskBits)
+            fracGood = np.sum(good)/len(good.ravel())
```

Comment:
You could save some characters and nanoseconds by using `good.size()` instead of `len(good.ravel())`.

Reply:
Looks like it's just `.size` though (attribute, not method).